Entries/Copy-X/Copy-X.md

```diff
@@ -0,0 +1,306 @@
+
+The ring protection present on all copy-X discs will cause errors during dumping. This will not cause issues with the playability of the final image, but may make it difficult or slow to copy/rip data, and makes it very difficult to guarantee you've gotten a 1:1 rip of all data on the disc.
+
+copy-X light: None. Copying all files from the disc besides the dummy data directory is sufficient to defeat the protection.
+
+copy-X professional: Any sort of basic copy of the disc image appears to work. An ISO of the game will function, as will Bin/Cue or other similar formats if the game has audio tracks.
+
+*** 
+
+## Associated File Attributes:
+
+### File Directories:
+
+#### "System":
+Directory for dummy files used by most professional[^48393][^DE_EM2DX] discs so far, as well as some light[^Kenny] discs.
+
+#### "ZDAT"/"zdata":
+ZDAT is the directory for dummy files used by some[^84759][^107929][^81628][^101786], but not all[^Kenny] light discs, and two known professional[^Petterson] disc so far. One calls it zdata[^108150], one calls it ZDATA[^108150], and one uses System[^Kenny] instead. Further variance likely.
+
+### File Names:
+
+Note: file names are not case sensitive, and vary from disc to disc.
+
+#### "[m/g]ov_*.x64":
+Dummy file associated with professional. Names include gov_io.x64[^108150][^Petterson], gov_05.x64[^48393][^DE_EM2DX], and mov_io.x64.
+
+#### "iofile.x64"[^48393][^DE_EM2DX][^108150][^Petterson]:
+Dummy file associated with professional. Seems to be directly referenced by optgraph.dll.
+
+#### "sound.x64"[^48393][^108150][^Petterson]:
+Dummy file associated with professional. Seemingly not present on at least one[^DE_EM2DX] professional disc.
+
+#### "optgraph.dll"[^DE_EM2DX][^48393][^108150][^Petterson]:
+Dll file responsible for software protection on professional discs.
+
+***
+
+## Known Manufacturing Information:
+
+### Known Manufacturing Plants:
+
+* `optimal media production`[^CopyX_2009_Document]
+
+### Known Ringcode Information:
+
+#### Mastering Code:
+
+Note: Tivola discs add "Tivola    MM/[last 2 digits of year]" in between "manufactured by optimal media production" and "A######-0[Disc Number]", but otherwise follow that format.
+
+* `manufactured by optimal media production    A######-0[Disc Number]`[^108150] (The majority of known samples have this mastering code. Only the ones that don't are listed below.)
+* `OH! Datenservice GmbH   www.oh-datenservice.de    A######-0[Disc Number]`[^48393][^DE_EM2DX] (Seemingly mainly used by Sixteen Tons Entertainment)
+* `KIDDINX    A######-0[Disc Number][^Petterson]
+
+#### Mastering SID Code:
+
+* `IFPI L571`(Every known sample)
+
+#### Toolstamp or Mastering Code:
+
+* N/A
+
+#### Mould SID Code:
+
+*  `IFPI 97xx`(Every known sample)
+
+***
+
+# Reference Material:
+
+## Media: 
+
+1. 'SoftICE for Windows 95' being detected[^108150]:  
+!["SoftICE für Windows95 aktiv!"](SoftICE_95_Detected.png "'SoftICE for Windows 95' being detected")
```

Comment:
Please make the first description slightly more descriptive, for example, the first one should be``"SoftICE-Test" + "SoftICE für Windows95 aktiv!" + "OK"``.

Reply:
Fixed.